### ISSUE TITLE
Facility Locator pagination Prev/Next links do not scroll to top

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#14895]
## Description
When using the Prev/Next links on the pagination on [Facility Locator](https://www.va.gov/find-locations), the screen should load with the top of the results in view so the veteran can find their place at the top of the list on the next or previous page. Focus should also be at the "Showing X - XX of XX results" so folks using assistive technology can easily orient themselves to the new list of results.

Currently, the behavior scrolls to the bottom of the page and the pagination numbers are at the top of the view.

See [this Slack comment](https://dsva.slack.com/archives/C52CL1PKQ/p1691772540955969?thread_ts=1691683090.666189&cid=C52CL1PKQ) for video example.

The pagination & focus behavior should match [Find a Form](https://www.va.gov/find-forms/?q=disability) paging.

### User story
AS A Veteran
I WANT the page to scroll to the top when I use Prev/Next links in paginated lists
SO THAT I can easily find my place in the list on the next page

Slack thread for context on [related issue in VAMC Events](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/14755): https://dsva.slack.com/archives/C52CL1PKQ/p1691683090666189

## Acceptance criteria
- [ ] When the user activates Prev/Next links, the page should load with the top of the results list in view (as Find a Form currently does)
- [ ] Focus should go to "Showing XX - XX of XX results" when the user activates Prev/Next links (as Find a Form currently does)
- [ ] Accessibility review

**Comment by xiongjaneg** [comment]
Currently, it's an odd interaction. Laura is noticing a quick jump to the footer and then back to the top of the page.
